fidesctl/src/fideslang/models.py

```diff
@@ -289,16 +340,32 @@ class DatasetMetadata(BaseModel):
 class Dataset(FidesModel):
     "The Dataset resource model."
 
-    meta: Optional[Dict[str, str]]
-    data_categories: Optional[List[FidesKey]]
+    meta: Optional[Dict[str, str]] = Field(
+        description="An optional object that provides additional information about the Dataset. You can structure the object however you like. It can be a simple set of `key: value` properties or a deeply nested hierarchy of objects. How you use the object is up to you: Fides ignores it."
+    )
+    data_categories: Optional[List[FidesKey]] = Field(
+        description="Array of Data Category resources identified by `fides_key`, that apply to all collections in the Dataset.",
+    )
     data_qualifier: FidesKey = Field(
         default="aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified",
+        description="Array of Data Qualifier resources identified by `fides_key`, that apply to all collections in the Dataset.",
+    )
+    fidesctl_meta: Optional[DatasetMetadata] = Field(
+        description=DatasetMetadata.__doc__,
+    )
+    joint_controller: Optional[ContactDetails] = Field(
+        description=ContactDetails.__doc__,
+    )
+    retention: Optional[str] = Field(
+        default="No retention or erasure policy",
+        description="An optional string to describe the retention policy for a dataset. This field can also be applied more granularly at either the Collection or field level of a Dataset.",
+    )
+    third_country_transfers: Optional[List[str]] = Field(
+        description="An optional array to identify any third countries where data is transited to. For consistency purposes, these fields are required to follow the Alpha-3 code set in [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3).",
```

Comment:
I don't think this should have a markdown url, but leaving as a question for now 🤔

Reply:
ill check and see how it looks in the docs